crates/holochain/CHANGELOG.md

```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+- Init functions can now make zome calls. [#1186](https://github.com/holochain/holochain/pull/1186)
```

Comment:
as you noted on #1258 this addition doesn't belong here. it's duplicated from the most recent release. is this a case of pressing the "merge base into this branch" duplicating changelog items?

Reply:
```suggestion
```

I did a merge with the git cli tool, not sure how this happened... it must have made it look like that line should be there, regardless, I'll delete it.